### DISCUSSION TITLE
Week 1 Status Updates

### DISCUSSION THREAD

**Discussion by roryeiffe** [roryeiffe/Fresh-Results#6] [General]
Hi guys, since this is the first status update, reply to this with everything you've worked on since the project started up to March 25th. Here are the questions to keep in mind:

What did you work on this week?

What sort of technical blockers are you facing?

What are you going to work on next week?

What sort of new technologies did you learn, if any?

Put any links to commits/other contributions here.

Feel free to add more questions as you think of them!

**Comment by roryeiffe** [comment]
### What did you work on this week:

Up to now, I helped get the Github repository set up and created files such as licenses, code of conduct, readme, etc. I added all current team members as collaborators and created wikis to describe general information about the project, a start guide, and contribution guidelines. 

In terms of coding contributions, I worked with Abdul and Covenant to set up the initial codebase including a manifest.json which holds information about the extension itself as well as injecting React into the project. On Wednesday (3/25), I worked with Abdul, Serena, and Covenant to brainstorm ideas and create mockups of what sort of features we would like to add to the extension. Some of these features are filtering out search results, filtering out webpage content, changing the default color of the extension, and adding custom groups of keywords that the user would like to have blocked from their browsing experience. I also did a little bit of logo design. 

These designs can be found at https://www.figma.com/file/N6RD9WowT8LbuWFit0kE9f/Spoiler-Block

### What sort of technical blockers are you facing:

I'm not facing any significant technical blockers at this point. 

### What are you going to work on next week:

Next week, I will work on implementing the designs that we have created in Figma. 

### What sort of new technologies did you learn, if any:

I learned about how to set up a chrome extension. My learning progress is documented in the start guide wiki page:
https://github.com/roryeiffe/Spoiler-Block/wiki/Start-Guide

### Links to Contributions:

https://github.com/roryeiffe/Spoiler-Block/wiki
https://github.com/roryeiffe/Spoiler-Block/commit/e369b2f54d9f8f4babfa3b4ccd49a07c39339cf9
https://github.com/roryeiffe/Spoiler-Block/commit/f9bb18744ee0106d2443dd7c6aa7bc0a43d4e474
https://github.com/roryeiffe/Spoiler-Block/commit/fb364c1234f9dcb2acdcb009e3e02f2df16c01e6
https://github.com/roryeiffe/Spoiler-Block/wiki/Start-Guide
https://github.com/roryeiffe/Spoiler-Block/wiki/Contribution-Guidelines
https://www.figma.com/file/N6RD9WowT8LbuWFit0kE9f/Spoiler-Block?node-id=0%3A1
https://github.com/roryeiffe/Spoiler-Block/discussions/4
figma.com/file/N6RD9WowT8LbuWFit0kE9f/Spoiler-Block?node-id=30%3A9 (Pop-up and Logo Design)

**Comment by SerenaChen1** [comment]
During week 1, I mainly contributed by working on the UI team's Figma doc to create some mockups for different use cases. For example, I made the mockups for blocking spoilers on a webpage and the popup on a webpage that warns the user about the possibility of spoilers on that page. I wasn't able to attend the meeting where Rory, Abdul, and Covenant set up the initial codebase for the extension, but I made sure to read through the minutes and set it up on my own time before our next meeting.

**Comment by BreadGarLick** [comment]
For week one, I helped with the generating of ideas for the extension's function as well as general back-end brainstorming. I also created the initial backed word replacement code, for removing a single selected word.

I am attempting to become more familiar with javascript.

**Comment by TheCovenant** [comment]
### What did you work on this week:

In this week I worked on setting up the Github repo and setting up the starting point for the chrome extension.  This involved creating a manifest.json and adding React to the project. I learned how to test a chrome extension by unpacking the build and using developer mode.

### What sort of technical blockers are you facing:

I'm not facing any significant technical blockers at this point. 

### What are you going to work on next week:

Next week, I work on planning the scope of the UI team, dividing up tasks, and creating mock-ups

### What sort of new technologies did you learn, if any:

I learned about how to set up a chrome extension and also how to make use of React for creating Chrome extensions.

### Links to Contributions:
